composer.json

```diff
@@ -6,8 +6,8 @@
         "friendsofphp/php-cs-fixer": "~2.1"
     },
     "autoload": {
-        "psr-0": {
-            "Json": "src/"
+        "psr-4": {
+            "Json\\": "src/Json/"
```

Comment:
Can you explain why you'd need `\\`? I can't find anything indicating that it's required
https://stackoverflow.com/questions/24868586/what-are-the-differences-between-psr-0-and-psr-4
https://www.php-fig.org/psr/psr-4/

Reply:
Ah, found it
https://getcomposer.org/doc/04-schema.md#psr-4

> Namespace prefixes must end in \\ to avoid conflicts between similar prefixes. For example Foo would match classes in the FooBar namespace so the trailing backslashes solve the problem: Foo\\ and FooBar\\ are distinct.